bin/v-add-user-ssh-jail

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# info: add user ssh jail
+# options: USER [RESTART]
+#
+# example: v-add-user-ssh-jail admin
+#
+# This function enables ssh jailed environment
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Argument definition
+user=$1
+restart=$3
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+chroot="/srv/jail/$user"
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+# Checking if jailkit is installed
+if [ ! -x /sbin/jk_init ]; then
+	exit
+fi
+
+check_args '1' "$#" 'USER'
+is_format_valid 'user'
+
+# Perform verification if read-only mode is enabled
+check_hestia_demo_mode
+
+#----------------------------------------------------------#
+#                       Action                             #
+#----------------------------------------------------------#
+
+# Get shell full path
+shell_path=$(grep "^$user:" /etc/passwd | cut -f 7 -d :)
+
+# Set home folder permission to root
+if [ -d "/home/$user" ]; then
+	chown root:root /home/$user
+fi
+
+add_chroot_jail "$user"
+
+# Installing shell files into the user chroot directory
+# - IMPORTANT - MODIFY THE FOLLOWING LINES AND THE FILE jk_init.ini ACCORDING TO YOUR SYSTEM AND YOUR PREFERENCES
+/sbin/jk_init -f -j $chroot extendedshell netutils ssh sftp scp git php php5_6 php7_0 php7_1 php7_2 php7_3 php7_4 php8_0 php8_1 php8_2 > /dev/null 2>&1
+/sbin/jk_cp -f -j $chroot /bin/id > /dev/null 2>&1
+
```

Comment:
After first time running this command I did get:

```
usermod: user eris is currently used by process 75716
failed to execute usermod -d /srv/jail/eris/./home/eris -s /usr/sbin/jk_chrootsh eris
failed to modify user eris
usermod: no changes
usermod: no changes
```
This might be caused that I was already logged in into via ssh

How ever everything I try to login I get the error: 

Last login: Sun Nov  5 21:53:04 2023 from xxxxx
-bash: groups: command not found

Reply:
This indeed has to do that you're already logged in. There is nothing to be done in that situation but honestly I don't think it will happen often and can be easily fixed by changing the shell to unjailed and then to jailed again.

Reply:
Still this error happens even after enabling / disabling it ...

Last login: Sun Nov 5 21:53:04 2023 from xxxxx
-bash: groups: command not found

Reply:
This is because we're unable to change the passwd and groups file within the jail and thus cannot assign the right homedir and shell within the jail.

The only solution I can think of is disconnecting the user if we change the jail if there is a current session.

Reply:
> This is because we're unable to change the passwd and groups file within the jail and thus cannot assign the right homedir and shell within the jail if there is an active ssh session.
> 
> The only solution I can think of is disconnecting the user if we change the jail if there is a current session.

This was without an active ssh session (Except for the root user as well)

Reply:
Ok, I will check it out.

Reply:
This should be fixed now. I added groups and made sure all groups are set before jailing the user